src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,8 +36,9 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        User user = userService.findByEmail(authentication.getName()).get();
```

Comment:
let's throw an exception in case user is not found, the same for other similar cases